apps/meteor/server/models/raw/Users.js

```diff
@@ -3079,4 +3079,17 @@ export class UsersRaw extends BaseRaw {
 			},
 		);
 	}
+
+	countActiveUsersWithCustomRoles() {
+		const customRoleRegex = /^[0-9a-fA-F]{24}$/;
```

Comment:
Can u put a sample of what this regex is supposed to match?

Reply:
Sure, I'll add a comment, but this regex matches object IDs from MongoDB. Default roles are created with an user friendly ID.